setup.py

```diff
@@ -2,18 +2,18 @@
 
 from setuptools import setup
 
-setup( name = "pyxdg",
-       version = "0.28",
-       description = "PyXDG contains implementations of freedesktop.org standards in python.",
-       maintainer = "Freedesktop.org",
-       maintainer_email = "xdg@lists.freedesktop.org",
-       url = "http://freedesktop.org/wiki/Software/pyxdg",
-       packages = ['xdg'],
-       classifiers = [
-            "License :: OSI Approved :: GNU Lesser General Public License v2 (LGPLv2)",
-            "Programming Language :: Python :: 2.7",
-            "Programming Language :: Python :: 3",
-            "Topic :: Desktop Environment",
-       ],
+setup(
```

Comment:
Most of the diff is because `black` auto-reformats files on edit for me; I can back this out if it is a problem. The `python_requires` line is the relevant part.

Reply:
In general, if a project isn't already auto-formatted, I'd ask contributors to turn off their autoformatter when working on it, to avoid changing unrelated lines. The changes here are small enough that I won't ask you to revert them, though.